javaslang/src/main/java/javaslang/collection/Map.java

```diff
@@ -151,6 +151,28 @@ default boolean contains(Tuple2<K, V> element) {
     }
 
     /**
+     * If the specified key is not already associated with a value,
+     * attempts to compute its value using the given mapping
+     * function and enters it into this map.
```

Comment:
`this` map is unchanged anyway. Wouldn't the use of `this` reserved word be misleading?